cosqq.html

```diff
@@ -5,95 +5,132 @@
+            <div class="logo">
+              <img src="public/icons/AM-system.png" alt="logo"/>
+            </div>
+            <div class="nav">
+              <nav>
+                <ul>
+                  <li>
+                    <a href="">About</a>
+                  </li>
+                  <li>
+                    <a href="">Offer</a>
+                  </li>
+                  <li>
+                    <a href="">Contact</a>
+                  </li>
+                </ul>
+              </nav>
+            </div>
         </div>
-        <img src="images/photo1.png">
-      </header>
+    </header>
 
+  <main>      
+    
+    <section id="hero">
+      <div class="welcome_text">
+        <h1>Lorem ipsum dolor </h1>
+        <h2>Lorem ipsum dolor sit amet consectetur adipisicing elit.Lorem ipsum dolor sit amet consectetur adipisicing elit.Lorem ipsum dolor sit amet consectetur adipisicing elit.</h2>
+        <input class="subbmit_explore" type="submit" value="Explore"/>
+      </div>
+      <div class="start_image">
+        <img src="public/images/undraw_House_searching_re_stk8 1.png" alt="house">
+      </div>
+  </section>
+  
+  <div class="triangle-bottomright"></div>
 
-      <section class="offer clearfix">
-        <h1>Offer</h1>
-        <h2>Lorem ipsum dolor sit amet consectetur adipisicing elit.</h2>
-        <div class="prod">
-          <img src="images/services1.png" alt="box">
-          <h2>Lorem ipsum</h2>
-          <p class="description">Lorem ipsum dolor sit amet, architecto asperiores, sequi obcaecati, alias sunt ea. Soluta, rerum, perspiciatis.</p>
-          <button>Lern More</button>
-        </div>
-        <div class="prod">
-          <img src="images/services2.png" alt="box">
-          <h2>Lorem ipsum</h2>
-          <p class="description">Lorem ipsum dolor sit amet, architecto asperiores, sequi obcaecati, alias sunt ea. Soluta, rerum, perspiciatis.</p>
-          <button>Lern More</button>
-        </div>
-        <div class="prod">
-          <img src="images/services3.png" alt="box">
-          <h2>Lorem ipsum</h2>
-          <p class="description">Lorem ipsum dolor sit amet, architecto asperiores, sequi obcaecati, alias sunt ea. Soluta, rerum, perspiciatis.</p>
-        <button>Lern More</button>
-        </div>
-      </section>
-
+    <section id="offer"> 
+      <h1>Offer</h1>
+      <h2>Lorem ipsum dolor sit amet consectetur adipisicing elit.</h2>
+      <div class="all_products">
+          <div class="products"> 
+            <div class="offer_tile_image">
+              <img src="public/icons/Vector.png" alt="box"> 
+            </div>
+            <h2>Lorem ipsum</h2>
+            <p>Lorem ipsum dolor sit amet, architecto asperiores, sequi obcaecati, alias sunt ea. Soluta, rerum, perspiciatis.Lorem ipsum dolor sit amet, architecto asperiores, sequi obcaecati, alias sunt ea. Soluta, rerum, perspiciatis.</p>
+            <input class="button1" type="submit" text="Lern More"/>
```

Comment:
dont use numbers in class naming, except if you declare variables with global color/fonts etc.

Reply:
additional you have typo LEARN! not lern